### ISSUE TITLE
Mention the exact compatible ansible version(s)

### ISSUE THREAD

**Issue by JocelynDelalande** [ginsys/ansible-role-seafile#22]
As far as now, from my experience, the role only works with ansible 1.9. (tested unsuccessfuly with 2.x)

**Comment by srvg** [comment]
Yes, because not really tested etc.. I'd love to get 2.x compat, but that might depend on #19, which is stalled because some ansible core bug where the cron module fails within docker...

**Comment by JocelynDelalande** [comment]
The issue is not that it's not compatible with all ansible versions, but that this fact is not mentioned anywhere :-)

**Comment by srvg** [comment]
Yes, I know, just saying why it's still stuck there. And I would rather focus on getting it on par with v2.

Though not a full answer to your request, there's a hint at https://github.com/ginsys/ansible-role-seafile/blob/stable/meta/main.yml#L9

**Comment by JocelynDelalande** [comment]
> Though not a full answer to your request, there's a hint at
> https://github.com/ginsys/ansible-role-seafile/blob/stable/meta/main.yml#L9

The missing information is the max version :-)

**Comment by srvg** [comment]
Which is why I just commented that here https://github.com/ansible/ansible/issues/18375#issuecomment-258616340 :-)
